playbooks/openshift/post-install.yml

```diff
@@ -1,4 +1,9 @@
 ---
+
+# Post-Install tasks for specific providers
+- import_playbook: gcp/post-install.yaml
```

Comment:
should be `gcp/post-install.yml`

Reply:
@etsauer all my files are with the yml extension...

Reply:
@raffaelespazzoli I now see a mix of `.yml` and `.yaml` ... from before, we have standardized on `.yml`, so let's continue using this shorter version